tests/component/test_export_signal_properties.py

```diff
@@ -2,62 +2,81 @@
+
+
+def test__export_signal__get_enum_property__returns_default_value(ai_voltage_task: Task):
     """Test to validate getter for export signal property of enum type."""
-    assert ai_voltage_chan_task.export_signals.samp_clk_output_behavior == ExportAction.PULSE
+    assert ai_voltage_task.export_signals.samp_clk_output_behavior == ExportAction.PULSE
 
 
-def test__export_signal__set_enum_property__returns_assigned_value(ai_voltage_chan_task: Task):
+def test__export_signal__set_enum_property__returns_assigned_value(ai_voltage_task: Task):
     """Test to validate setter for export signal property of enum type."""
     value_to_set = ExportAction.LEVEL
-    ai_voltage_chan_task.export_signals.samp_clk_output_behavior = value_to_set
+    ai_voltage_task.export_signals.samp_clk_output_behavior = value_to_set
 
-    assert ai_voltage_chan_task.export_signals.samp_clk_output_behavior == value_to_set
+    assert ai_voltage_task.export_signals.samp_clk_output_behavior == value_to_set
 
 
-def test__export_signal__reset_enum_property__returns_initial_value(ai_voltage_chan_task: Task):
+def test__export_signal__reset_enum_property__returns_default_value(ai_voltage_task: Task):
     """Test to validate resetting export signal property of enum type."""
-    ai_voltage_chan_task.export_signals.samp_clk_output_behavior == ExportAction.INTERLOCKED
+    ai_voltage_task.export_signals.samp_clk_output_behavior == ExportAction.INTERLOCKED
 
-    del ai_voltage_chan_task.export_signals.samp_clk_output_behavior
+    del ai_voltage_task.export_signals.samp_clk_output_behavior
 
-    assert ai_voltage_chan_task.export_signals.samp_clk_output_behavior == ExportAction.PULSE
+    assert ai_voltage_task.export_signals.samp_clk_output_behavior == ExportAction.PULSE
 
 
-def test__export_signal__get_string_property__returns_value(ai_voltage_chan_task: Task):
+def test__export_signal__get_string_property__returns_default_value(ai_voltage_task: Task):
     """Test to validate getter for export signal property of string type."""
-    assert ai_voltage_chan_task.export_signals.start_trig_output_term == ""
+    assert ai_voltage_task.export_signals.start_trig_output_term == ""
 
 
 def test__export_signal__set_invalid_routing_destination__throws_daqerror(
-    ai_voltage_chan_task: Task,
+    ai_voltage_task: Task,
 ):
     """Test to validate setter for export signal property of string type."""
-    ai_voltage_chan_task.export_signals.start_trig_output_term = "RTSI"
+    ai_voltage_task.export_signals.start_trig_output_term = "RTSI"
 
-    with pytest.raises(DaqError) as e:
-        _ = ai_voltage_chan_task.export_signals.start_trig_output_term
-    assert e.value.error_type == DAQmxErrors.INVALID_ROUTING_DESTINATION_TERMINAL_NAME_ROUTING
+    with pytest.raises(DaqError) as exc_info:
+        _ = ai_voltage_task.export_signals.start_trig_output_term
+    assert (
+        exc_info.value.error_type == DAQmxErrors.INVALID_ROUTING_DESTINATION_TERMINAL_NAME_ROUTING
+    )
```

Comment:
The error is really caused by the set call, but it isn't reported until you verify the task. Instead of using a get call to verify the task, use task.control(TaskMode.VERIFY)

I think the set and verify should both be in the act section.

Reply:
Used `ai_voltage_task.control(TaskMode.TASK_VERIFY)` instead of set call.